### PR TITLE
Refactor and clean code

### PR DESCRIPTION
**Pull request by pmareke**
This commit refactors a little bit the Vue code as we talked before.

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -96,11 +98,11 @@ export default class Index extends Vue {
 
   @spotify.Action('fetchCurrentSong') public fetchCurrentSong!: Function;
 
-  public get showLogin() {
```
Comment (SantiMA10) [inline pages/index.vue:99]: Is there any reason to remove `public` from getters and methods?

**Review by SantiMA10 — APPROVED** [review]

**Comment by SantiMA10** [comment]
@pmareke only one question before merging it. Thanks for your contribution!

File: pages/index.vue (hunk 1 above)
Comment (pmareke) [reply pages/index.vue:99]: I think is not necessary due the fact that is code execute inside the component.

The question for me is, why did you make the method public?

Comment (SantiMA10) [reply pages/index.vue:99]: I supposed that is needed to be able to access it from the component, but thanks for the information.

**Comment by SantiMA10** [comment]
@all-contributors please add @pmareke for maintenance and code

**Pull request merged**